index.d.ts

```diff
@@ -524,6 +533,11 @@ declare module '@primer/components' {
   export interface DialogProps extends CommonProps, LayoutProps, Omit<React.HTMLAttributes<HTMLDivElement>, 'color'> {
     isOpen: boolean
     onDismiss: () => unknown
+    narrow?: boolean
+    wide?: boolean
```

Comment:
Should these be a `variant` instead? So that a consumer can't pass `narrow={true} wide={true}`.

Reply:
I considered that, but I usually like to reserve the use of variant for groupings of styles that affect more than 1 css property - I figured it would be pretty obvious that using both `narrow` and `wide` at the same time wouldn't work so decided to leave it as is!

Reply:
👍  - good to know there is a rule of thumb on `variant` vs separate props.